src/components/Modal/Modal.vue

```diff
@@ -525,6 +527,32 @@ export default {
 				this.slideshowTimeout.clear()
 			}
 		},
+		/**
+		 * Get List of focusableelmets from content container
+		 */
+		getFocusableList() {
+			const contentContainer = this.$refs.mask.querySelector('.modal-container')
+			const focusableList = contentContainer.querySelectorAll(
```

Comment:
I think now you don't need to query all, just the first element ?
basically rename the function to "getFirstFocusableElement" ? (assuming the focus trap doesn't do this already automatically)

Reply:
...i do not longer need  getFocusableList at all. forgot to remove it.